client/layout/guided-tours/tours/boosted-post-tour.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { overEvery as and } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ButtonRow,
+	LinkQuit,
+	makeTour,
+	Quit,
+	Step,
+	Tour,
+} from 'layout/guided-tours/config-elements';
+import {
+	isAbTestInVariant,
+	isEnabled,
+} from 'state/ui/guided-tours/contexts';
+import { isDesktop } from 'lib/viewport';
+
+export const BoostedPostTour = makeTour(
+	<Tour
+		name="boostedPostSurvey"
+		path="/posts/"
+		version="20170127"
+		when={ and(
+			isEnabled( 'boosted-post-survey' ),
+			isDesktop,
+			isAbTestInVariant( 'boostedPostSurvey', 'enabled' )
+		) }
```

Comment:
@mcsf We changed our approach for this away from immediately after a post is published to gaging interest in all of the posts as they view them. Either option is fine for our needs so it makes sense to to go with an option that requires a bit less overhead to implement it.